taskcluster/async/asyncclient.py

```diff
@@ -144,7 +143,7 @@ class AsyncBaseClient(BaseClient):
                 if response.status == 204:
                     return None
 
-            except aiohttp.HttpProcessingError as rerr:
+            except aiohttp.ClientError as rerr:
```

Comment:
The docs suggest to me that it's `aiohttp.ClientResponseError` here?

http://aiohttp.readthedocs.io/en/stable/client_reference.html#aiohttp.ClientResponse.raise_for_status